packages/pieces/pastebin/src/lib/actions/create-paste.ts

```diff
@@ -0,0 +1,69 @@
+import { Property, createAction } from "@activepieces/pieces-framework";
+import { makeClient, pastebinCommon } from "../common";
+import { PasteExpiry, PastePrivacy } from "../common/client";
+
+export default createAction({
+    name: 'create_paste',
+    displayName: 'Create Paste',
+    description: 'Creates a new paste',
+    props: {
+        authentication: pastebinCommon.authentication(true),
```

Comment:
![image](https://github.com/activepieces/activepieces/assets/1812998/828897dd-4eb6-4252-a772-91d58b8d479a)

I added the following, but not sure why username and password is required.

Can you help me with that?

Reply:
it's not required it's optional and only needed if you want to create private pastes

Reply:
I also tried to move username and password into a different credential set but it seems like only a single credential set per piece is allowed right now since it will show me the dev token credential in the selection for the username/password credential so I moved it into a single one with optional props

Reply:
Added note, good point!